lexicons/tools/ozone/moderator/addUser.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.moderator.addUser",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Add a user as moderator in the ozone service.",
```

Comment:
Would be good docs to mention in each of these descriptions what privileges are required to use the endpoint. Eg, all require auth, but maybe `listUsers` only requires an ozone user of any role?